master/reference/advanced/etcd-rbac/kubernetes.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Setting up Kubernetes with Calico components utilizing etcd RBAC
+---
+
+## Overview
+
+When using etcd with RBAC all components that access etcd must be configured
+with the proper certificates.  Here are the different components that need to be
+configured and some help with that task for operating Kubernetes with Calico
+accessing an etcd with RBAC.
+
+## Components that need etcd Roles
+
+The following components need certificates with a Common Name that matches an
+etcd user that has been given appropriate roles allowing access to the paths
+listed or linked below.
+- kube-apiserver
```

Comment:
Maybe include this in the "paths" doc ... so that would mean the paths doc would need to include non Calico paths.

Or a separate paths doc for "known" non-Calico paths.